source/_integrations/fujitsu_anywair.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: Fujitsu anywAIR
+description: Instructions on how to integrate Fujitsu anywAIR controller into Home Assistant.
+ha_category:
+  - Climate
+  - Cover
+  - Light
+  - Sensor
+  - Update
+ha_release: 2024.1
```

Comment:
This cannot be correct. The virtual integration always need to match the version of introduction of the parent integration.